seed/challenges/basic-javascript.json

```diff
@@ -1817,6 +1823,7 @@
         "    // Only change code above this line.",
         "    ",
         "    if (slotOne === slotTwo && slotTwo === slotThree) {",
+		"	 	$('.logger').html('It's A Win');",
         "      return slotOne;",
```

Comment:
There is a secondary issue that in the "spaces" before this line there are two tab characters which break the parser.  Those will need to be removed as well as the quotes being fixed.